js/foam/u2/DAOListView.js

```diff
@@ -18,13 +18,26 @@
 CLASS({
   package: 'foam.u2',
   name: 'DAOListView',
-  extends: 'foam.u2.Element',
+  extends: 'foam.u2.View',
   requires: [
     'foam.u2.DetailView'
   ],
+  imports: [
```

Comment:
It's accessed as `this.selection` in some places. The `this.X.selection$` check is there to see if selection is enabled at all. Wiring up the click listener is a waste of time otherwise.